### PR TITLE
Add `jax.numpy.argsort` to list of unsupported jax operations

### PR DESCRIPTION
**Pull request by josh146**

### CODE REVIEW

**Review by dime10 — APPROVED** [review]
Thanks Josh! Can you also open GH/SC issue?

**Comment by josh146** [comment]
Yep done! https://github.com/PennyLaneAI/catalyst/issues/810

**Review by isaacdevlugt — APPROVED** [review]
🍠

File: doc/dev/sharp_bits.rst [hunk 1]
```diff
@@ -510,6 +510,7 @@ that doesn't work with Catalyst includes:
 
 - ``jax.numpy.polyfit``
```
Comment (erick-xanadu) [inline doc/dev/sharp_bits.rst:511]: I thought Romain added support for `polyfit` via custom calls?

Comment (josh146) [reply doc/dev/sharp_bits.rst:511]: Nice catch, I wasn't aware. If so we could remove this.

Comment (isaacdevlugt) [reply doc/dev/sharp_bits.rst:511]: Indeed! This works:

```
>>> x = jnp.array([1.2, 0.1, 2.7, 0.6])
>>> y = jnp.array([0.2, 0.4, 0.6, 0.8])
>>>
>>> @qml.qjit
... def fn(x):
...     return jnp.polyfit(x, y, deg=4)
...
>>> fn(x)
array([-0.0059191 ,  0.01930699,  0.01312885,  0.09407333, -0.22579063])
```

Comment (isaacdevlugt) [reply doc/dev/sharp_bits.rst:511]: I updated https://app.shortcut.com/xanaduai/epic/65875?cf_workflow=500000005&ct_workflow=all&group_by=none&vc_group_by=day

**Pull request merged**